maps/rivers_and_bridges.go

```diff
@@ -374,3 +376,147 @@ var riversAndBridgesExtraLargeHazards = []rules.Point{
 	{X: 8, Y: 12},
 	{X: 12, Y: 16},
 }
+
+type IslandsAndBridgesMediumHazardsMap struct{}
+
+func (m IslandsAndBridgesMediumHazardsMap) ID() string {
+	return "hz_islands_bridges"
+}
+
+func (m IslandsAndBridgesMediumHazardsMap) Meta() Metadata {
+	return Metadata{
+		Name:        "hz_islands_bridges",
+		Description: `Creates fixed maps that have a lake of hazard in the middle with rivers going in the cardinal directions and around the edges of the map. Bridges across the rivers are provided at key points`,
+		Author:      "Battlesnake",
+		Version:     1,
+		MinPlayers:  1,
+		MaxPlayers:  4,
+		BoardSizes:  FixedSizes(Dimensions{11, 11}),
+		Tags:        []string{TAG_FOOD_PLACEMENT, TAG_HAZARD_PLACEMENT, TAG_SNAKE_PLACEMENT},
+	}
+}
+
+func (m IslandsAndBridgesMediumHazardsMap) SetupBoard(lastBoardState *rules.BoardState, settings rules.Settings, editor Editor) error {
+	if !m.Meta().BoardSizes.IsAllowable(lastBoardState.Width, lastBoardState.Height) {
+		return rules.RulesetError("This map can only be played on a 11x11 board")
+	}
+
+	err := setupRiverAndBridgesBoard(islandsAndBridgesMediumStartPositions, islandsAndBridgesMediumHazards, lastBoardState, settings, editor)
+	if err != nil {
+		return err
+	}
+	return nil
```

Comment:
Not blocking - these can just be:
```
return setupRiverAndBridgesBoard(...)
```

Reply:
good point.  my Go ignorance is really showing there